src/Pipeline.cpp

```diff
@@ -507,7 +507,7 @@ const std::map<std::string, JITExtern> &Pipeline::get_jit_externs() {
     return contents->jit_externs;
 }
 
-void Pipeline::add_custom_lowering_pass(IRMutator2 *pass, void (*deleter)(IRMutator2 *)) {
+void Pipeline::add_custom_lowering_pass(IRMutator2 *pass, std::function<void()> deleter) {
```

Comment:
why no have the deleter take the `IRMutator2*` as a parameter?

Reply:
It seemed cleaner to have it argumentless.